spec/rest/defaults.test.js

```diff
@@ -36,7 +36,7 @@ define(['ably', 'shared_helper'], function(Ably, helper) {
 		test.deepEqual(normalisedOptions.fallbackHosts, Defaults.FALLBACK_HOSTS);
 		test.equal(normalisedOptions.tls, true);
 
-		test.deepEqual(Defaults.getHosts(normalisedOptions), [normalisedOptions.host].concat(Defaults.FALLBACK_HOSTS));
+		test.deepEqual(Defaults.getHosts(normalisedOptions), [normalisedOptions.host].concat(Defaults.FALLBACK_HOSTS.slice(0,3)));
```

Comment:
Again, simply slicing effectively means we only ever use `a-c.ably-realtime.com` which is not the intention